src/servers/src/http/greptime_result_v1.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use axum::response::{IntoResponse, Response};
+use axum::Json;
+use common_error::ext::ErrorExt;
+use common_error::status_code::StatusCode;
+use common_query::Output;
+use common_recordbatch::util;
+use reqwest::header::HeaderValue;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::http::error_result::ErrorResponse;
+use crate::http::{GreptimeQueryOutput, HttpRecordsOutput, HttpResponse};
+
+pub(crate) const GREPTIME_V1_TYPE: &str = "greptime_v1";
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub struct GreptimedbV1Response {
+    // deprecated - backward compatible
+    r#type: &'static str,
+    code: u32,
```

Comment:
`type` is introduced in v0.5 only by `#[serde(tag = "type")]`.

Since we now have "X-GreptimeDB-Format" header, I tend to remove this field at all.

Also `code` is to convey error code in the previous code and we don't add it for InfluxDBV1Response. I tend to remove it because a non-error response always has code = 0.

Reply:
I think we can just delete the `type`.

Reply:
Updated.